app/Tenant.php

```diff
@@ -189,4 +189,8 @@ private function getPaystackPayload(array $options = []) {
 
     return $payload;
   }
+
+   public function has_bank_account(){
+    return ($this->defaultBankAccount() ? TRUE : FALSE);
```

Comment:
```suggestion
    return ($this->defaultBankAccount() ? true : false);
```

dont think they need to be uppercase